### PR TITLE
Cheminformatics deploy

### PR DESCRIPTION
**Pull request by bedroesb**
I annotated the workflows linked in the tutorials and tested them in a container if they open correctly if they are installed the way it is described in the documentation.

The container linked in the documentation will be updated today with the corresponding workflows.

### CODE REVIEW

File: cheminformatics/deploy/all_Covid_tools.yaml [hunk 1]
```diff
@@ -0,0 +1,148 @@
+install_repository_dependencies: true
+install_resolver_dependencies: true
+install_tool_dependencies: true
+tools:
+- name: enumerate_charges
+  owner: bgruening
+  revisions:
+  - 2a868592ebcb
+  tool_panel_section_label: Covid-19
```
Comment (bgruening) [inline cheminformatics/deploy/all_Covid_tools.yaml:9]: Should we have a Cheminformatics category and a genomics one?

File: cheminformatics/deploy/workflows/Cheminformatics-4-TransFS_scoring.ga [hunk 2]
```diff
@@ -0,0 +1,302 @@
+{
```
Comment (bgruening) [inline cheminformatics/deploy/workflows/Cheminformatics-4-TransFS_scoring.ga:1]: @bedroesb there are wokflows in each sub-folder, can you remove them? It does not make sense to have them in two places.

Comment (bedroesb) [reply cheminformatics/deploy/workflows/Cheminformatics-4-TransFS_scoring.ga:1]: I will compare them to make sure I have the right ones and than delete them

File: cheminformatics/deploy/all_Covid_tools.yaml (hunk 1 above)
Comment (bedroesb) [reply cheminformatics/deploy/all_Covid_tools.yaml:9]: Good idea, should covid-19 still be mentioned ?

Comment (bgruening) [reply cheminformatics/deploy/all_Covid_tools.yaml:9]: Your call, your baby :)

**Comment by bedroesb** [comment]
@bgruening I found out that the workflows in this github repo where newer than the ones where the tutorials link to (galaxy europe user workflows). So I've taken the new ones from this repo off course.

**Comment by bgruening** [comment]
@bedroesb thanks a lot.

**Pull request merged**